static/src/stylesheets/module/commercial/_creatives.scss

```diff
@@ -1,5 +1,7 @@
 .creative__tracking-pixel {
     position: absolute;
+    width: 1px;
```

Comment:
I don't. The class was there before my PR. I just removed two presentation attributes.